ios/Podfile

```diff
@@ -70,6 +70,8 @@ target 'iNaturalistReactNative' do
         config.build_settings['HEADER_SEARCH_PATHS'] << '"$(PODS_CONFIGURATION_BUILD_DIR)/React-bridging/react_bridging.framework/Headers" '
       end
     end
+    #react_native_post_install(installer)
+    `sed -i -e  $'s/__IPHONE_10_0/__IPHONE_12_0/' Pods/RCT-Folly/folly/portability/Time.h`
```

Comment:
Is this necessary? If so can you explain why in a comment?

Reply:
It's explained [here](https://stackoverflow.com/questions/70357696/error-in-pods-rct-folly-folly-portability-time-h-and-typedef-redefinition-with-d) and [here](https://github.com/facebook/react-native/issues/31480):
But basically you get a compilation error for iOS when building it using latest XCode and iOS Build target, and this was the only workaround I found